### ISSUE TITLE
Unexpected error

### ISSUE THREAD

**Issue by reiz** [versioneye/veye-checker#4]
I just created a inventory.csv file with that tool. As soon I tried to resolve the SHAs to prod_keys I got this error message: 

```
Processing sha response....
Failed to get product details for /Users/reiz/workspace/nedap/extlib/cactus13/cactus-13-1.7.2/cactus-ant-1.7.2.jar, 1ce0e6cbe720b3ff5239f01bd439f4e89972186c
Going to checkup product by SHA: 05dfad7bbf4cecff888478e67e6792f146e65324
Processing sha response....
Going to check product details by matched SHA result
```

**Comment by reiz** [comment]
The API response for that SHA looks like this: 

```
 {
    "language": "Java",
    "prod_key": "commons-httpclient/commons-httpclient",
    "version": "2.0.2",
    "group_id": "commons-httpclient",
    "artifact_id": "commons-httpclient",
    "classifier": null,
    "packaging": "jar",
    "prod_type": "Maven2",
    "sha_value": "48856bef6c64eb9689eab125197fdefc399c7858",
    "sha_method": "sha1"
  }
```

**Comment by reiz** [comment]
Here another one: 

```
[
  {
    "language": "Java",
    "prod_key": "cargo/cargo",
    "version": "0.5",
    "group_id": "cargo",
    "artifact_id": "cargo",
    "classifier": null,
    "packaging": "jar",
    "prod_type": "Maven2",
    "sha_value": "05dfad7bbf4cecff888478e67e6792f146e65324",
    "sha_method": "sha1"
  }
]
```

**Comment by timgluz** [comment]
Found issue: it failed to parse the product details when there was no license url;

**Comment by reiz** [comment]
Good Job!
